packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -487,11 +490,8 @@ const SavedChartsHeader: FC = () => {
                                             }
                                         />
                                     ) : null}
-                                    {userCanManageCharts && (
-                                        <PostHogFeature
```

Comment:
Remove feature flag component because it sets a `div`. Used the feature flag hook instead.